TeslaMateAgile/Data/Options/PGEOptions.cs

```diff
@@ -0,0 +1,23 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace TeslaMateAgile.Data.Options;
+
+public class PGEOptions
+{
+    [Required]
```

Comment:
Is everything in here required or are some optional? Currently all are required except the `Program` field